sssd_test_framework/utils/sssd.py

```diff
@@ -832,6 +832,7 @@ def local(self) -> None:
             proxy_lib_name="files",
```

Comment:
I don't insist but maybe it makes sense to have this `local` call the self.proxy() method instead with the appropriate arguments, which already sets the default_domain

Reply:
I intend to rework the proxy provider stuff, so this is an interim fix.